extensions/mssql/src/ui/dialogBase.ts

```diff
@@ -20,6 +20,9 @@ export const DefaultMinTableRowCount = 1;
 const TableRowHeight = 25;
 const TableColumnHeaderHeight = 30;
 
+export const tableHeader = { 'border-color': '#ccc', 'color': '#666666', 'font-weight': 'normal' };
+export const tableRow = { 'border-color': '#ccc' };
```

Comment:
the border color is the one used in tables in the sql projects extension, but can change it if a different color is preferred.